### PR TITLE
SDKS-3420 Use sample environment for Sample App.

### PR DESCRIPTION
**Pull request by witrisna**
# JIRA Ticket

[SDKS-3420](https://bugster.forgerock.org/jira/browse/SDKS-3420)

# Description

SDKS-3420 Use sample environment for Sample App.

### CODE REVIEW

File: samples/app/src/main/java/com/example/app/env/EnvViewModel.kt [hunk 1]
```diff
@@ -13,59 +13,107 @@ import androidx.compose.runtime.mutableStateOf
...
+        servers.add(PingAM)
+        servers.add(PingAdvancedIdentityCloud)
+        servers.add(PingOne)
+        servers.add(PingFederate)
     }
 
     fun select(context: Context, options: FROptions) {
-        if(options.server.url.contains("pingone")) {
+        // Discover settings from the specified PingOne server's .well-known endpoint
+        if(options.server.url.contains("pingone") || options.server.url.contains("pingfederate")) {
```
Comment (forgerock-chris) [inline samples/app/src/main/java/com/example/app/env/EnvViewModel.kt:104]: As PingFed is self-hosted, the chance of the domain containing "pingfederate" is very low.

In addition to the "pingone" check, could we have a separate `if` to check for `.well-known/openid-configuration`, and use `discover` if found?

**Review by forgerock-chris — DISMISSED** [review]
Thanks for adopting these changes over Andy. I'll update my tutorials to make the steps for editing the sample a lot simpler.

File: samples/app/src/main/java/com/example/app/env/EnvViewModel.kt (hunk 1 above)
Comment (forgerock-chris) [reply samples/app/src/main/java/com/example/app/env/EnvViewModel.kt:104]: To clarify a bit, in the instructions I'll say something along the lines of:
-  Enter the .well-known URI of your PingFederate server.
If that URL _does not_ contain the word "pingfederate", the sample won't work "out of the box", and I'll need an additional instruction to make sure that "discover" is used.
Hope that makes sense?

**Review by spetrov — DISMISSED** [review]

File: samples/app/src/main/java/com/example/app/env/EnvViewModel.kt (hunk 1 above)
Comment (witrisna) [reply samples/app/src/main/java/com/example/app/env/EnvViewModel.kt]: I have used the instance instead:
```
if(options === PingOne || options === PingFederate) {
```

Comment (forgerock-chris) [reply samples/app/src/main/java/com/example/app/env/EnvViewModel.kt]: Okay, I'll update the PingFederate instructions to match, cheers Andy.
I see this is heading for `develop`, will it also go to `main`, or only when there's a new release?

Comment (witrisna) [reply samples/app/src/main/java/com/example/app/env/EnvViewModel.kt]: Only when there is a new release.

**Review by forgerock-chris — APPROVED** [review]

**Review by spetrov — APPROVED** [review]

**Pull request merged**